### PR TITLE
[wip] Remove deadcode

### PR DESCRIPTION
**Pull request by alexzielenski**
Removes unused/dead validation code

### CODE REVIEW

File: pkg/validation/validate/slice_validator.go [hunk 1]
```diff
@@ -23,15 +23,14 @@ import (
...
-	AdditionalItems *spec.SchemaOrBool
-	Items           *spec.SchemaOrArray
-	KnownFormats    strfmt.Registry
-	Options         SchemaValidatorOptions
+	Path         string
+	In           string
+	MaxItems     *int64
+	MinItems     *int64
+	UniqueItems  bool
+	Items        *spec.Schema
```
Comment (apelisse) [inline pkg/validation/validate/slice_validator.go:31]: Why are you turning this schema list into a single schema?

**Review by apelisse — COMMENTED** [review]
Thanks for removing that code! Just one question.

**Comment by apelisse** [comment]
Do we still need this or was it merged as part of another PR?

**Comment by Jefftree** [comment]
@alexzielenski is this still needed?

**Pull request closed without merging**